tests/unit/authors/test_authors_tasks.py

```diff
@@ -89,6 +89,24 @@ def test_new_ticket_context(data, extra_data, user):
     assert ctx['user_comment'] == 'Foo bar'
 
 
+def test_new_ticket_context_handles_unicode(user):
+    data = {'name': {'preferred_name': u'Diego Martínez'}}
+    extra_data = {}
+
+    obj = MockObj(data, extra_data)
+
+    expected = {
+        'object': obj,
+        'email': 'foo@bar.com',
+        'subject': u'Your suggestion to INSPIRE: author Diego Martínez',
+        'user_comment': '',
+    }
+    result = new_ticket_context(user, obj)
+    print("Resulting object: ", result)
```

Comment:
No worries, while you are working on a PR you can always squash your commits and `push -f` so no need for the extra commit.